docs/Playing/how-to-play.rst

```diff
@@ -1,224 +1,366 @@
-
-.. tip:: Try to build on horses or near fish. These and other tile resources are your friend.
-
-Now that you have a city or two, you'll want to set the science rate as high as your government type will
-allow.  Don't worry about the tax rate, since you won't be building any improvements to drain your cash;
-you'll be building :unit:`settlers`. Every city should be churning out :unit:`settlers`. The more
-:unit:`settlers` you make, the more cities you can have; the more cities you have, the faster you gain tech;
-the faster you gain tech, the faster you win. After you have built as many cities as your corner of the world
-will hold, turn the :unit:`settlers` to irrigating and building roads until you can build :unit:`workers`.
-
-.. note:: If the food surplus in a city drops to +1 from supporting too many settlers, and you can't
-    rearrange people to increase it, then switch them to building :improvement:`temples`. Unless you make
-    contact with another player, don't worry about building military units just yet.
-
-All this time, you have been gaining techs as fast as possible. What you should be shooting for is first
-:strong:`The Republic`, then :strong:`Democracy`, then :strong:`Railroad`, and then
-:strong:`Industrialization`. Some people go for :strong:`Monarchy` before :strong:`The Republic`.  As soon
-as you've researched a new government type, start a revolution and change over to it. Cities operate much
-better as Republics than they do under Despotism, but note that it's much harder to keep military units
-outside of city limits under a Republic. Also, don't forget to recheck your rates after you've changed
+First of all, this isn't a *perfect* strategy, but it will get you started playing Freeciv21. Part of the
+appeal of Freeciv21 is in developing new strategies as you play more games. This guide is mostly based upon
+the Classic ruleset. There are other rulesets out there that ship with the game, but the base information
+presented here is the same for all rulesets.
+
+The game is divided into several steps or stages:
+
+#. Choosing the first city site.
+#. Selecting your first city locations.
+#. Selecting your technological advance.
+#. Mapping the countryside.
+#. Defending the cities.
+#. Deciding which units to build first.
+#. Improving the land.
+#. Deciding where to build cities.
+#. Taking care of the cities.
+#. Interacting with other players (diplomacy).
+#. Exploring the world.
+#. Things to keep in mind.
+#. Production phase.
+#. Annihilation of your opponents.
+
+Choosing the First City Site
+----------------------------
+
+Start the game by wandering around :strong:`briefly` to find a good place to build the first city. Do not feel
+tempted to investigate any yellow roofed villages yet, they might contain :unit:`Barbarian` tribes. Build the
+city close to Resources (fish, wheat, wine, or buffalo) and perhaps close to fresh water (River or Lake). Keep
```

Comment:
Should the resources in parentheses be capitalized?

Reply:
Yep, good catch.